src/QMCWaveFunctions/BsplineFactory/SplineC2ROMPTarget.cpp

```diff
@@ -392,28 +387,32 @@ void SplineC2ROMPTarget<ST>::mw_evaluateDetRatios(const RefVectorWithLeader<SPOS
                                                   std::vector<std::vector<ValueType>>& ratios_list) const
 {
   assert(this == &spo_list.getLeader());
-  auto& phi_leader      = spo_list.getCastedLeader<SplineC2ROMPTarget<ST>>();
-  const size_t nw       = spo_list.size();
-  const size_t orb_size = phi_leader.size();
+  auto& phi_leader            = spo_list.getCastedLeader<SplineC2ROMPTarget<ST>>();
+  auto& mw_mem                = *phi_leader.mw_mem_;
+  auto& det_ratios_buffer_H2D = mw_mem.det_ratios_buffer_H2D;
```

Comment:
I'd rather save the vertical space than horizontal but that's your call.

Reply:
In this case, I prefer bring phi_leader.mw_mem_->det_ratios_buffer_H2D down to det_ratios_buffer_H2D.